proxylib/Sai.cpp

```diff
@@ -551,6 +551,20 @@ sai_status_t Sai::queryStatsCapability(
     return SAI_STATUS_NOT_IMPLEMENTED;
 }
 
+sai_status_t Sai::queryStatsStCapability(
+    _In_ sai_object_id_t switchId,
+    _In_ sai_object_type_t objectType,
+    _Inout_ sai_stat_st_capability_list_t *stats_capability)
+{
+    SWSS_LOG_ENTER();
+
+    SWSS_LOG_ERROR("not implemented, FIXME");
+
+    // TODO: Fix me
```

Comment:
comment not needed if error message contains the same

Reply:
Removed